app/AboutAppearanceViewController.m

```diff
@@ -24,6 +25,12 @@ @implementation AboutAppearanceViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+//    __block AboutAppearanceViewController *_self = self;
+//    [NSNotificationCenter.defaultCenter addObserverForName:@"updateStatusBar" object:nil queue:[NSOperationQueue mainQueue] usingBlock:^(NSNotification * _Nonnull note) {
+//        [_self prefersStatusBarHidden];
+//        NSLog(@"updateStatusBar: %d", _self.prefersStatusBarHidden);
+//        [_self setNeedsStatusBarAppearanceUpdate];
+//    }];
```

Comment:
You can just delete code we don't need anymore, it's all in git

Reply:
Lol that was code I was working on that I forgot to remove on the final commit. I'll take that gross monstrosity out